BUFR_TableD_en_07.csv

```diff
@@ -124,6 +124,73 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 07,Surface report sequences (land),307022,(Ground-based GNSS data),,202000,Change scale,Cancel,,Operational
 07,Surface report sequences (land),307022,(Ground-based GNSS data),,201000,Change data width,Cancel,,Operational
 07,Surface report sequences (land),307022,(Ground-based GNSS data),,015011,Log10 of integrated electron density,,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,301150,WIGOS identifier,,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,001015,Station or site name,,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,001040,Processing centre ID code,GNSS processing centre,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,008021,Time significance,= 30 Time of occurrence,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,301011,"Year, month, day",,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,301013,"Hour, minute, second",,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,301022,"Latitude/longitude (high accuracy), height of station",,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,010036,Geoid undulation,"Geoid height above WGS-84 ellipsoid, ±150 m to 1 cm",,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,025061,Software identification and version number,"GNSS processing software, softwareversion",,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,010004,Pressure,,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,012001,Temperature/air temperature,,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),,013003,Relative humidity,,,Operational
+07,Surface report sequences (land),307024,(Ground-based GNSS data - Slant Total Delay),Zenith Total Delay,120000,Delayed replication of 20 descriptors,,,Operational
```

Comment:
Verify that the subtitles "Zenith Total Delay" and "Slant Total Delay will not appear in the manual.